client/assets/js/board.js

```diff
@@ -27,4 +37,17 @@ export default class Board {
 		this.apples.push(apple);
 	}
 
+	getAvailableColor(){
+		let snakeColor = this.snakes.map(function(snake){
+			return snake.color;
+		});
+
+		let availableColor =  this.color.find(function(c) {
```

Comment:
This `find` call may return undefined. The `undefined`color will be passed to the snake :-/